scality_manila_utils/utils.py

```diff
@@ -119,6 +119,22 @@ def process_check(process):
                                    "started".format(process))
 
 
+def fsync_path(path):
+    """
+    Fsync a file or directory.
+
+    :param path: path to file or directory to fsync
+    :type path: string (unicode)
+    """
+    fd = None
+    try:
+        fd = os.open(path, os.O_RDONLY)
```

Comment:
I'd specialize this procedure to be used on directories, and not even support syncing of files: syncing a file should be done by the writer (who has an FD available already). Sync-after-close of a file could hide some bugs (even though that scenario isn't currently used, luckily).

As such, I'd consider updating the docstring and use
```python
fd = os.open(path, os.O_RDONLY | os.DIRECTORY)
```
here.